etc/profile-m-z/rymdport.profile

```diff
@@ -0,0 +1,62 @@
+# Firejail profile for rymdport
+# Description: Encrypted sharing of files, folders, and text between devices
+# This file is overwritten after every install/update
+# Persistent local customizations
+include rymdport.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+
+noblacklist ${HOME}/.config/fyne
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/fyne
+whitelist ${HOME}/.config/fyne
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
```

Comment:
```suggestion
#mkdir ${HOME}/.config/fyne
#whitelist ${HOME}/.config/fyne
#whitelist ${DOWNLOADS}
#include whitelist-common.inc
```

The PR enables the profile by default on firecfg and the program appears to be
intended to be used on arbitrary user files.